docs/index.asciidoc

```diff
@@ -337,3 +337,30 @@ NOTE: Client identity is not typically validated using SSL because the receiving
 include::{include_path}/{type}.asciidoc[]
 
 :default_codec!:
+
+==== Elasticsearch output
```

Comment:
```suggestion
[id="plugins-{type}s-{plugin}-es-output-notes"]
==== Using {esf-name} with the Elasticsearch output
```

Reply:
Suggest adding more words for better SEO performance and context in search results